### PR TITLE
enable ephemeral pins for cephfs tests

### PR DESCRIPTION
**Pull request by batrick**

### CODE REVIEW

File: misc/test-clone-rm-kernel.sh [hunk 1]
```diff
@@ -42,6 +42,9 @@ function do_clone_rm_kernel {
 
     mkdir -p /cephfs/destinations
     pushd /cephfs/destinations
+    if [ "$MAX_MDS" = d ]; then
```
Comment (sidharthanup) [inline misc/test-clone-rm-kernel.sh:45]: I don't think I understood this. Should it be `== d` ?

File: scripts/cephfs-git-clone.sh [hunk 2]
```diff
@@ -97,7 +97,7 @@ function do_test {
     ans -m shell -a "ceph fs set cephfs max_mds $max_mds" mon-000
     run ans --module-name=copy --args="src=misc/test-clone-rm-kernel.sh dest=/root/ owner=root group=root mode=755" clients
     pretest "$D" clients
-    run ans -m shell -a "/root/test-clone-rm-kernel.sh $(( count > 32 ? count/32 : 1 )) $max_mds" "$(nclients "$count" 32)"
+    run ans -m shell -a "/root/test-clone-rm-kernel.sh $(( count > NUM_CLIENTS ? count/NUM_CLIENTS : 1 )) d" "$(nclients "$count" NUM_CLIENTS)"
```
Comment (sidharthanup) [inline scripts/cephfs-git-clone.sh:100]: What is d here?

Comment (batrick) [reply scripts/cephfs-git-clone.sh:100]: d is just short hand for indicating to use distributed pins

File: misc/test-clone-rm-kernel.sh (hunk 1 above)
Comment (batrick) [reply misc/test-clone-rm-kernel.sh:45]: No, see `test(1)`.

Comment (sidharthanup) [reply misc/test-clone-rm-kernel.sh:45]: Thanks! My Bash is rusty.

File: scripts/cephfs-git-clone.sh (hunk 2 above)
Comment (sidharthanup) [reply scripts/cephfs-git-clone.sh:100]: Ahh got it. Thanks

**Comment by sidharthanup** [comment]
@batrick Won't it again try to apply ceph.dir.pin here: https://github.com/batrick/ceph-linode/blob/edcd9c037be3a79d4ab1f3286dbcdb39b39e9c9e/misc/test-clone-rm-kernel.sh#L16 , since MAX_MDS is not an empty string. Maybe also check if its not equal to `d` ?

**Comment by batrick** [comment]
> , since MAX_MDS is not an empty str

good point, I updated it to be cleaner. Havne't tested yet though.

**Pull request closed without merging**